app/views/posts/all.html.erb

```diff
@@ -0,0 +1,5 @@
+<h3> All posts</h3>
+
+<% @posts.each do |post| %>
+  <p><%= simple_format(post.message)%> - <%= post.created_at.to_date%></p>
```

Comment:
I have amended this script to show the post in order to have date and text all in one line.
Have a look at my pull request.